package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/main.js",
   "scripts": {
     "start": "node lib/main.js",
-    "build": "tsc --project tsconfig.prod.json"
+    "build": "tsc --project tsconfig.prod.json",
+    "test": "jest"
   },
   "author": {
     "name": "Sumiran Aggarwal",
```

Comment:
Shouldn't the author be Microsoft Corporation as is in tasks?

Reply:
It is same as other actions. We mention individual's name under author.